classes/Mail.php

```diff
@@ -987,4 +987,12 @@ protected static function dieOrLog(
             $die
         );
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected static function decryptSMTPPass($encryptedPW)
+    {
+        return openssl_decrypt($encryptedPW,"AES-128-CTR", "PrestaShopSMTP", 0, '3922847190294561');
```

Comment:
I think that all these values (`PrestaShopSMTP` and `3922847190294561`) should be stored in constants or global configuration.

Reply:
Yes, this i can agree on. Will do it when i have time.